cmd2/argparse_completer.py

```diff
@@ -332,7 +335,8 @@ def consume_flag_argument() -> None:
             """Consuming token as a flag argument"""
             # we're consuming flag arguments
             # if this is not empty and is not another potential flag, count towards flag arguments
-            if token and token[0] not in self._parser.prefix_chars and flag_action is not None:
+            # if the token is a single character length, it doesn't matter that it matches a flag prefix
```

Comment:
(Pedantic) This might be clearer as "If the token is a single character, it doesn't matter whether it matches a flag prefix"